components/button/Button.js

```diff
@@ -120,11 +120,22 @@ const factory = (ripple, FontIcon) => {
         'data-react-toolbox': 'button',
       };
 
-      return React.createElement(element, props,
+      const renderElementWrapper = element === 'button' && others.onMouseEnter && this.props.disabled;
```

Comment:
We shouldn't filter by element type `button` since the same problem happens with links too.

Reply:
I don't see the same happening for links. Could you give a code snippet showing that?